integration-tests/validation.py

```diff
@@ -187,13 +187,21 @@ def validate_status_messages(model):
     auth = requests.auth.HTTPBasicAuth(user, password)
     resp = await asyncify(requests.get)(url, auth=auth, verify=False)
     assert resp.status_code == 200
+
     # get k8s version
     app_config = await model.applications['kubernetes-master'].get_config()
     channel = app_config['channel']['value']
-    version_string = channel.split('/')[0]
-    k8s_version = tuple(int(q) for q in re.findall("[0-9]+", version_string)[:2])
-    # dashboard will present a login form prompting for login
-    if (k8s_version < (1, 8)):
+    # if we do not detect the version from the channel eg edge, stable etc
+    # we should default to the letest dashboard url format
+    version_detected = False
+    k8s_version = (2, 0)
+    if '/' in channel:
+        version_string = channel.split('/')[0]
+        k8s_version = tuple(int(q) for q in re.findall("[0-9]+", version_string)[:2])
+        # dashboard will present a login form prompting for login
+        version_detected = True
+
+    if version_detected and k8s_version < (1, 8):
```

Comment:
If I've read this correctly, you don't need the `version_detected` var at all, since you default `k8s_version` to be greater than (1, 8).

Reply:
Habits i've picked up from more verbose languages. Easy fix.